website-1.x/core/Showcase.js

```diff
@@ -7,15 +7,6 @@
 
 const React = require('react');
 const PropTypes = require('prop-types');
-const users = require('../data/users');
-
-users.forEach((user) => {
-  if (!user.image.startsWith('/img/users')) {
-    throw new Error(
-      `User image should be self-hosted in /img/users folder. This was not the case for ${user.image}`,
-    );
-  }
-});
```

Comment:
seen this check just after, but does not. look like the appropriate place to do it.